config.yaml

```diff
@@ -56,6 +56,11 @@ params:
       img: images/content_images/case_studies/sports.jpg
       alttext: Cricket ball on green field.
       url: /case-studies/cricket-analytics
+    - title: Posture Estimation using DNN 
```

Comment:
I tried to tweak this title, but there's so little space......

Two minor things: I can't find a use of "posture" on the DeepLabCut, only "pose". And "DNN" I'm not a huge fan of, "deep neural networks" isn't a term used by deep learning practitioners anymore I think. Just "neural networks" is enough (I searched the TensorFlow and PyTorch websites to confirm).

`Pose estimation using neural networks` doesn't fit. `Pose estimation using deep learning` does though - what do you think about that title?